### ISSUE TITLE
issues with --pass

### ISSUE THREAD

**Issue by psiho** [dhleong/ps4-waker#120]
I'm not sure if I'm missing something obvious but behavior of ps4-waker is quite unpredictable for me. Here are a few scenarios. I have PS4 with 2 different accounts using it behind passcodes. I'm trying to wake it up and set it to standby using CLI. All is connected well and credentials created.

*** Scenario 1 - PS4 in standby
`ps4-waker`
... PS4 is turned on, but ps4-waker returns error: "Login error: Passcode is required". PS4 is on "press PS button screen"
... also ps4-waker does not exit. I have to stop it with CTRL+C

*** Scenario 2 -PS4 is in status above (waiting for PS button to be pressed to login)
`ps4-waker standby`
...gives me error:
```
Unexpected error Error: Failed to login: PASSCODE_IS_NEEDED (22)
    at errorFromResult (/usr/lib/node_modules/ps4-waker/dist/device.js:493:16)...
```

*** Scenario 3 - PS4 is in standby
... lets say passoced on PS is "up up up up":
`ps4-waker --pass 2222` 
... no change from secnario 1

*** Scenario 4 - PS4 is in status above (waiting for PS button to be pressed to login)
`ps4-waker --pass 2222 standby` 
... no change from secnario 2

*** Scenario 5 - PS4 is in status above (waiting for PS button to be pressed to login)
`ps4-waker --skip-login standby`
ps4-waker exits with message "WAKEUP sent to device... 192.168.1.20" but nothing happens.

What works is:
1) if PS4 is ON and my user is logged in. Then this puts it to standby:
`ps4-waker standby`

2) if PS4 is in STANDBY then this wakes it up properly and ps4-waker exits properly:
`ps4-waker --skip-login`

In general, I'm trying to use this in Node-red on OpenHAB2 but with ps4-waker not exiting properly and with --pass option ignored it's close to unusable for me. But it seems too much stuff is wrong for a simple bug, I'm suspecting something is wrong on my side

**Comment by dhleong** [comment]
Interesting. I refactored the CLI ops handling a while ago, so it's possible the `--pass` flag is actually dropped; I'm surprised nobody has reported it until now, though, but I'll look into it.

`--skip-login` is incompatible with `standby` (you have to be logged in to issue `standby`) but it shouldn't hang. I can add checks for incompatible flags like that.

Thanks for the report! I may have some time tomorrow to check this out.

**Comment by psiho** [comment]
Just downgraded to 1.5.0 and it works ok there

**Comment by dhleong** [comment]
Released 1.8.6 with a fix for this, and a handful of other fixes I discovered along the way. Let me know if you have any other issues!
